src/applications/edu-benefits/complaint-tool/helpers.js

```diff
@@ -73,7 +73,7 @@ function pollStatus(guid, onDone, onError) {
         if (!res || res.data.attributes.state === 'pending') {
           pollStatus(guid, onDone, onError);
         } else if (res.data.attributes.state === 'success') {
-          onDone(res.data.attributes.response);
+          onDone(res.data.attributes.parsed_response);
```

Comment:
Using parsed_response rather than response here, as indicated in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11784